.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
         // cause problems working with the Samples Browser.
         "MD003": {"style": "atx"},
         "MD013": false
-    }
+    },
+    "python.pythonPath": "/usr/local/bin/python"
```

Comment:
This makes sense; my only concern is that this would override any locally set Python path (e.g.: for Windows users, or for users running in a conda env).

Reply:
Sorry about that, this was meant to go with the Docker environment, that should be the same for everyone regardless of the platform! I'm happy to take it out again if you think this is going to be an issue?

Reply:
Ah, sorry, when I reviewed this I also was thinking of this as Docker-specific for some reason, but this is just the general VS Code settings file for anyone who clones the repo. Thanks for catching that, @cgranade.

Reply:
No worries! As a future PR (not a huge rush), it may be worth moving that to `.devcontainer/devcontainer.json`, as [the `settings` property can be used to override VS Code settings](https://code.visualstudio.com/docs/remote/containers#_default-container-specific-settings) from within that devcontainer while leaving them otherwise unmodified.